File: myapp/parent/Step40_Parallel_Assember.cs
using sfn_ut.sfn.parallel;

namespace sfn_ut.myapp.parent;

public class Step40_Parallel_Assember : SfnParallelResultsAssembler<ParentPayload> {
  public override ParentPayload Execute(ParentPayload original, List<ParentPayload> parallelOutputs) {
    original.ParallelA = parallelOutputs[0].ParallelA;
    original.ParallelB = parallelOutputs[1].ParallelB;

    return original;
  }
}

File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/parent/Step40_Parallel_Assember.java
package com.sfnut.myapp.parent;

import com.sfnut.sfn.parallel.SfnParallelResultsAssembler;
import java.util.List;

public class Step40_Parallel_Assember extends SfnParallelResultsAssembler<ParentPayload> {
    @Override
    public ParentPayload execute(ParentPayload original, List<ParentPayload> parallelOutputs) {
        original.setParallelA(parallelOutputs.get(0).getParallelA());
        original.setParallelB(parallelOutputs.get(1).getParallelB());

        return original;
    }
}
